datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -662,15 +666,17 @@ public void onPartitionsRevoked(Collection<TopicPartition> topicPartitions) {
       }
     }
 
-    updateConsumerAssignment(_consumer.assignment());
+    Set<TopicPartition> newAssignment = new HashSet<>(_consumerAssignment);
```

Comment:
Sorry, this part of change is no longer needed as _consumer.assignment will be the source of truth since we are doing task level locking. I will remove this change.